### PR TITLE
Jsonize rules

### PR DESCRIPTION
**Pull request by Ian-Stewart-Binks**
Note - the details are comprised of both arrays of strings and strings themselves. We have a nested list structure here. This is bad.

### CODE REVIEW

**Comment by Anirudh94** [comment]
@Ian-Stewart-Binks why is this bad? It gives a nice nested structure doesn't it?

**Comment by Ian-Stewart-Binks** [comment]
@Anirudh94 It does, but note that the nested list is associated with the previous string.

File: util/foosball_rules.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
+    " Winning conditions": [
```
Comment (Anirudh94) [inline util/foosball_rules.json]: nit: space between the " and the 'W'

File: util/foosball_rules.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        [
+            " If the ball is knocked out, it is placed at the mids of the team who lost the last point."
+        ],
+    "Dead ball":
+        [
+            " Both teams must agree that a ball is dead before any player touches the ball.",
+            " If the ball is dead and near the center, it is placed at the mids of the team who lost the last point.",
+            " If the ball is ahead of a teams forwards, it is placed at the nearest corner."
+        ],
+    " Beers":
```
Comment (Anirudh94) [inline util/foosball_rules.json]: also here

**Comment by Anirudh94** [comment]
Anyways, patch looks good to me. There are some inconsistencies with style related issues but rather than point it out manually I say we merge and then add a json linting grunt task

File: util/foosball_rules.json (hunk 2 above)
Comment (Ian-Stewart-Binks) [inline util/foosball_rules.json]: Thanks.

**Pull request closed without merging**